scripts/accumulator.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/716d723547fc7212c70b4612441ac483ab055cdd